tutorials/script/core/10_multiple_fallback_nodes.py

```diff
@@ -0,0 +1,160 @@
+# %% [markdown]
+"""
+# Core: 10. Multiple fallback nodes
+
+This tutorial shows basics of creating a separate fallback node for every flow. In this case it is done via `LOCAL` nodes.
+
+Let's do all the necessary imports from DFF:
+"""
+
+# %pip install dff
+
+
+"""Importing all the necessary modules:"""
+
+# %%
+from dff.pipeline import Pipeline
+from dff.script import TRANSITIONS, RESPONSE, Message, LOCAL
+import dff.script.conditions as cnd
+
+# %% [markdown]
+"""
+Let's create a script that will consist of several flows, that can be accessed through dialogue:
+"""
+
+# %%
+toy_script = {
+    "greeting_flow": {
+        "start_node": {
+            RESPONSE: Message(),
+            TRANSITIONS: {
+                ("greeting_flow", "say_hi_node"): cnd.true()
+            }
+        },
+
+        "say_hi_node": {
+            RESPONSE: Message(text="Hello, how can I help you?"),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"): cnd.exact_match(Message(text="auth")),
+                ("information_flow", "response_node"): cnd.exact_match(Message(text="info"))
+            }
+        },
+
+        "fallback_node": {
+            RESPONSE: Message(text="Unknown Command. Try again."),
+            TRANSITIONS: {"say_hi_node": cnd.exact_match(Message(text="hi"))
+            }
+        }
+    },
+    "authorization_flow": {
+        LOCAL: {
+            TRANSITIONS: {
+                ("auth_fallback_node"):
+                    cnd.true()
+            }
+        },
+        "response_node": {
+            RESPONSE: Message(text="Write your name, please."),
+            TRANSITIONS: {
+                ("authorization_flow", "auth_password_node"):
+                    cnd.exact_match(Message(text="admin")),
+                ("authorization_flow", "auth_success_node"):
+                    cnd.exact_match(Message(text="John Doe"))
+            }
+        },
+
+        "auth_success_node": {
+            RESPONSE: Message(text="Authentication successful."),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"):
```

Comment:
Maybe it should restart the dialog (go to start node). It doesn't make much sense to ask for credentials over and over again.